package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "libp2p-circuit",
+  "version": "0.0.2",
+  "description": "JavaScript implementation of circuit/switch relaying",
+  "main": "src/index.js",
+  "scripts": {
+    "lint": "aegir-lint",
+    "build": "aegir-build",
+    "test": "aegir-test --env node",
+    "release": "aegir-release",
+    "release-minor": "aegir-release --type minor",
+    "release-major": "aegir-release --type major",
+    "coverage": "aegir-coverage",
+    "coverage-publish": "aegir-coverage publish"
+  },
+  "pre-commit": [
+    "lint",
+    "test"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/libp2p/js-libp2p-circuit.git"
+  },
+  "keywords": [
+    "IPFS"
+  ],
+  "author": "Dmitriy Ryajov <dryajov@gmail.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/libp2p/js-libp2p-circuit/issues"
+  },
+  "homepage": "https://github.com/libp2p/js-libp2p-circuit#readme",
+  "eslintConfig": {
+    "extends": [
+      "./node_modules/aegir/config/eslintrc.yml"
+    ],
+    "rules": {
+      "strict": "off"
+    }
+  },
```

Comment:
Please don't monkey patch our codestyle, we want to use the same for every single module in the same way.